.env.dist

```diff
@@ -22,13 +22,10 @@ APP_TEST_DB=ergonode_test
 
 ###> Xdebug Configuration ###
 # Default xdebug config for php target php_final_with_xdebug
-# This ip can be get by using shell command:
-# ifconfig | grep -A1 docker0 | grep inet
-# on Mac OS X please provide device IP number (en0)
 # if you have firewall you need enable listening on tcp port 9000
 # for ufw firewall you can use command like
 # sudo ufw allow 9000/tcp
-XDEBUG_CONFIG=remote_host=172.17.0.1
```

Comment:
`host.docker.internal`
Works only in Docker Desktop for Mac Or Windows. 

https://docs.docker.com/docker-for-windows/networking/
https://docs.docker.com/docker-for-mac/networking/

And there is no Docker Desktop for Linux.

You can update only doc. 

![Zaznaczenie_0403](https://user-images.githubusercontent.com/1453265/92384283-1ad29300-f110-11ea-9710-3c9461f5af76.png)

Reply:
Weirdo, was sure this was resolved on Linux already but well - reverted the change though left remote_connect_back to `false`. Without it, the remote_host value will be ignored.